### DISCUSSION TITLE
Actions is down, returning 500 responses

### DISCUSSION THREAD

**Discussion by sayhiben** [community/community#30860] [Actions] [answered]
`https://github.com/ORG/REPO/actions` returns a 500 response. Please prioritize ASAP.

<img width="713" alt="image" src="https://user-images.githubusercontent.com/678890/186784504-cc634d70-65d0-4d17-aaa0-840c813b0a7c.png">

**Comment by DenverCoder1** [comment, marked as answer]
It seems there was an incident that lasted about 20 minutes earlier today.

https://www.githubstatus.com/incidents/qjf5mzdcp335

As far as I can tell everything is currently operational.
